search/django/weedid/utils.py

```diff
@@ -107,7 +107,7 @@ def remove_entity_local_record(user_id, upload_id):
             os.remove(dir_path)
 
 
-def retrieve_listing_info(query_entity):
+def retrieve_listing_info(query_entity, awaiting_review):
```

Comment:
Now I'm wondering if this should available to staff at all times, not just when awaiting review? This is fine as it is.